lib/presentation/feedback/bloc/feedback_page_bloc.dart

```diff
@@ -1,12 +1,15 @@
+import 'package:appetizer/domain/repositories/feedback_repository.dart';
 import 'package:bloc/bloc.dart';
 import 'package:equatable/equatable.dart';
 
 part 'feedback_page_event.dart';
 part 'feedback_page_state.dart';
 
 class FeedbackPageBloc extends Bloc<FeedbackPageEvent, FeedbackPageState> {
-  FeedbackPageBloc()
-      : super(
+  final FeedbackRepository repo;
+  FeedbackPageBloc({
+    required this.repo,
+  }) : super(
           FeedbackPageState(
```

Comment:
implement new constructor FeedbackPageState.inital() and put the logic in the state.